app/server.py

```
@@ -27,78 +27,163 @@
+    url = f"{API_HOST}/v2/voices"
+    if query_string:
+        url = f"{url}?{query_string}"
 
     async with httpx.AsyncClient() as client:
-        response = await client.get(
-            f"{API_HOST}/v1/voices?model={model.value}",
-            headers=HTTP_HEADERS,
-        )
+        response = await client.get(url, headers=HTTP_HEADERS)
         if response.status_code != 200:
             raise Exception(f"Failed to get voices: {response.status_code}")
```

Ruff (0.14.13)

Create your own exception

(TRY002)